www/index.html

```diff
@@ -0,0 +1,1648 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="UTF-8">
```

Comment:
I think the indentation is fine, except for maybe the SVG? But I don't think it's as important for the SVG to have proper indentation as long as it isn't egregious, since it's generated code.